src/comp_dna.py

```diff
@@ -0,0 +1,17 @@
+def complementary_dna(dna):
+    comp_dna = ""
+    for char in dna:
+        if char == "C":
+            comp_dna = comp_dna + "G"
+        elif char == "T":
+            comp_dna = comp_dna + "A"
+        elif char == "G":
+            comp_dna = comp_dna + "C"
+        elif char == "A":
+            comp_dna = comp_dna + "T"
```

Comment:
Your approach is fine.

For education see if understand how these work:
```python
transcription_map = {"A": "T", "C": "G", "G": "C", "T": "A"}
comp_dna = ""
for char in dna:
    comp_dna += transcription_map[char]
```
or
```python
transcription_map = {"A": "T", "C": "G", "G": "C", "T": "A"}
comp_dna = "".join([ transcription_map[char] for char in dna])
```

Reply:
I tried to use this code, It didn't work.